### PR TITLE
Multiline text field component bug

### PR DESCRIPTION
**Pull request by chloe0592**
After discussion about the odd behaviour of multiline text field we've decided to remove the small resize button in the corner and it fixes the unexpected resizing on the "Enter" button.
Screenshot 2023-05-02 at 09 07 02

Other than that, we are hiding the "Extend/Collapse" button if there's not multiple rows in the `MultilineText` and `Translation` inputs.
We have't done that in the `RichText` inputs since that's currently too complex and it would require some extended refactoring.

**This closes:** [2201](https://github.com/commercetools/ui-kit/issues/2201).

### CODE REVIEW

File: packages/components/inputs/localized-multiline-text-input/src/translation-input.tsx [hunk 1]
```diff
@@ -122,10 +120,10 @@ const TranslationInput = (props: TranslationInputProps) => {
     if (onFocus) onFocus();
   }, [props.isCollapsed, onFocus, onToggle]);
 
-  // This checks if the content in the textarea overflows the minimum
-  // amount of lines it should have when collapsed
+  // This checks if the content in the textarea is greater than 38 (one row). If it is, then the toggle button will be shown.
```
Comment (CarlosCortizasCT) [inline packages/components/inputs/localized-multiline-text-input/src/translation-input.tsx:123]: I'm sorry but I'd like to ask about this so I can understand it better. What does 38 mean? Is it the number of characters that fill one row? Is it the height of the row? Is it another measurement?

Comment (chloe0592) [reply packages/components/inputs/localized-multiline-text-input/src/translation-input.tsx:123]: @CarlosCortizasCT sure, let me try to explain. If I understood everything correctly, [this](https://github.com/commercetools/ui-kit/blob/main/packages/components/inputs/localized-multiline-text-input/src/translation-input.tsx#L89) `handleHeightChange` is being referenced as a callback to the function `handleHeightChange` in [multiline-input.tsx](https://github.com/commercetools/ui-kit/blob/main/packages/components/inputs/input-utils/src/multiline-input/multiline-input.tsx#L40). 
Where we calculate the row count with dividing `scrollHeight` and `rowHeight` of the textarea. 
If it's only one row in the new theme the result will be number 38 and in the old theme it'll be 30.
It's somewhat complex written but when I console logged the `rowHeight` value I grasped it that it if we go above 38 it means it's not a single line anymore.
Screenshot 2023-05-02 at 13 12 11

Does it makes any more sense now? 😅

**Comment by FilPob** [comment]
looks good from my side :)

**Review by FilPob — APPROVED** [review]

File: packages/components/inputs/localized-multiline-text-input/src/translation-input.tsx (hunk 1 above)
Comment (CarlosCortizasCT) [reply packages/components/inputs/localized-multiline-text-input/src/translation-input.tsx:123]: Ok, thanks for the explanation.
So I understand 38 (in the new theme) in the height of the textarea in pixels as we can see in the browser inspector:
image

Looking at the numbers you shared, it matches the line-height of the component:
image

So at the end is seems like a complex way of calculating the height of the input.

The little drawback I see here is that we will have this 38 _magic number_ in our code which will work until we change the style of the components.
Perhaps we can calculate that number using the `line-height` and padding `styles` after its first render:
```js
const elementComputedStyles = getComputedStyle(<element_ref>);
const lineHeight = parseInt(elementComputedStyles.getPropertyValue('line-height'), 10); // -> 22
const paddingTop = parseInt(elementComputedStyles.getPropertyValue('padding-top'), 10); // -> 8
const paddingBottom = parseInt(elementComputedStyles.getPropertyValue('padding-bottom'), 10); // -> 8
const elementHeight = lineHeight + paddingTop + paddingBottom; // -> 38
```

What do you think?

Comment (chloe0592) [reply packages/components/inputs/localized-multiline-text-input/src/translation-input.tsx:131]: Yes, it absolutely makes more sense. Thank you for that! 🙇‍♀️ 
So I implemented it here - [b49e37c](https://github.com/commercetools/ui-kit/pull/2511/commits/b49e37c30737f23352f2636df740a067df423e2b). Let me know it there's something else.

**Comment by chloe0592** [comment]
@CarlosCortizasCT could I merge these changes?

File: packages/components/inputs/localized-multiline-text-input/src/translation-input.tsx [hunk 2]
```diff
@@ -122,10 +122,9 @@ const TranslationInput = (props: TranslationInputProps) => {
     if (onFocus) onFocus();
   }, [props.isCollapsed, onFocus, onToggle]);
 
-  // This checks if the content in the textarea overflows the minimum
-  // amount of lines it should have when collapsed
-  const contentExceedsShownRows =
-    contentRowCount > TranslationInput.MIN_ROW_COUNT;
+  // This checks if the content in the textarea is greater than 38 (one row). If it is, then the toggle button will be shown.
+  // This is to prevent the toggle button from showing when there is not enough content to expand/collapse.
+  // const contentExceedsShownRows = contentRowCount > firstRowHeight;
```
Comment (CarlosCortizasCT) [inline packages/components/inputs/localized-multiline-text-input/src/translation-input.tsx:127]: I think we should remove this comment block.

**Comment by CarlosCortizasCT** [comment]
> @CarlosCortizasCT could I merge these changes?

Please remember to add the changelog 🙏

File: packages/components/inputs/localized-multiline-text-input/src/translation-input.tsx (hunk 2 above)
Comment (chloe0592) [reply packages/components/inputs/localized-multiline-text-input/src/translation-input.tsx:127]: Yes, definitely. On it!

**Comment by chloe0592** [comment]
> > @CarlosCortizasCT could I merge these changes?
> 
> Please remember to add the changelog 🙏

Tnx for the reminder. 🙇‍♀️ Added here - [d526968](https://github.com/commercetools/ui-kit/pull/2511/commits/d5269680645f61fceb8c6fc389ae21872597a209)

File: packages/components/inputs/localized-multiline-text-input/src/translation-input.tsx (hunk 2 above)
Comment (chloe0592) [reply packages/components/inputs/localized-multiline-text-input/src/translation-input.tsx]: Removed here - [3a6cb5f](https://github.com/commercetools/ui-kit/pull/2511/commits/3a6cb5fab6d0fbb5880a1d41d4fe5b24024c2f78).

**Review by CarlosCortizasCT — APPROVED** [review]
Super nice! 🚀

**Pull request merged**